src/scripts/main.js

```diff
@@ -1 +1,65 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+const firstPromise = new Promise((resolve, reject) => {
+  body.addEventListener('mousedown', () => {
```

Comment:
use click
[difference between mousedown and click](https://stackoverflow.com/questions/19109754/difference-between-mousedown-and-click-in-jquery)